src/standalone/Soil/boundary_conditions.jl

```diff
@@ -203,7 +206,8 @@ function ClimaLSM.boundary_flux(
     p::NamedTuple,
     t,
 )::ClimaCore.Fields.Field
-    return bc.bc(p, t) .+ ClimaCore.Fields.zeros(axes(Δz))
+    FT = eltype(Δz)
+    return FT.(bc.bc(p, t)) .+ ClimaCore.Fields.zeros(axes(Δz))
```

Comment:
do we need this here? or will it get converted appropriately since it is added to a Field with (presumably) a fixed type?

Reply:
good point! I think it will get converted